woocommerce/woocommerce
# Fix typescript type errors in react admin `payments` & `payments-welcome`

### All Submissions:






### Changes proposed in this Pull Request:



Part of https://github.com/woocommerce/woocommerce/issues/32581

This PR fixes the  typescript type errors in react admin `payments` & `payments-welcome`.

To fix them

- I updated the types in `@woommcerce/data` & `<Pill />`
- Also, I added `@automattic/data-stores` to `@woocommerce/data` dev deps. I found it has many useful type utils that we need.

### How to test the changes in this Pull Request:

1. Run `pnpm nx build @woocommerce/data`
2. Observe the output of `pnpm tsc -b tsconfig.base.json | grep payments`


### Other information:




### FOR PR REVIEWER ONLY:
